meson.build

```diff
@@ -1,5 +1,5 @@
 project('rizin', 'c',
-  version: '0.2.0-git',
+  version: 'v0.2.0-git',
```

Comment:
I think this should be `0.2.0-git`, only tag names should be `v...` but can be discussed.

Reply:
Do you expect the tarball to contain rizin-0.2.0-git directory or rizin-v0.2.0-git?

Reply:
Doesn't matter I think. I find `v0.2.0` a bit nicer in the tarball since the file itself is called like that too but would be fine with both.

Reply:
so, if we use `v0.2.0-git`, the directory within the tarball is called `rizin-v0.2.0-git`. If we use `0.2.0-git` the directory is called `rizin-0.2.0-git`.Do you want to keep just the tag as `v{version}` and then all the artifacts as `{version}`(without the `v`)?

Reply:
No, I prefer `v0.2.0` for the artifact names and the folder inside of it. So unless having the `v` in the `project()` has any negative implications, I think the state in the pr here is good.